DateTimeMate.go

```diff
@@ -26,3 +26,35 @@ func convertRelativeDateToActual(from string) string {
 	}
 	return from
 }
+
+// shrinkPeriod convert a period into a brief period
+// only allow one replacement per each period
+// Ex: 1 hour 2 minutes 3 seconds => 1h2m3s
+// FIXME: almost redundant code for plural & singular -- try to fix with strings.NewReplacer
+func shrinkPeriod(period string) string {
```

Comment:
Maybe having a [map](https://github.com/jftuga/DateTimeMate/pull/5#discussion_r1683863526) could help later for example to write a code that will "expand"

1h3s to 1 hour and 3 seconds

Reply:
I'll assume you asking me for more details about my suggestion to be able to convert something short (brief) to its canonical form, as people could write it

64m3s => 1 hour, 4 minutes and 3 seconds

Something I think that cannot be done for the moment as you can only request to have something like "Hms" which would render to 1h4m3s

I'm suggesting that because your code provide a way to convert anything into a brief format from something long.

But you don't have the opposite to write 1 hour 4 minutes and 3 seconds